packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -201,6 +209,39 @@ function Preview( { item, categoryId, viewType } ) {
 	);
 }
 
+function Author( { item, viewType } ) {
```

Comment:
This component is copied verbatim from https://github.com/WordPress/gutenberg/blob/add/author-field-filter-template-parts/packages/edit-site/src/components/page-templates-template-parts/index.js#L117